src/Sulu/Component/Webspace/Analyzer/RequestAnalyzer.php

```diff
@@ -45,11 +51,25 @@ public function analyze(Request $request)
     }
 
     /**
+     * Initilizes the request attributes lazy.
+     *
+     * @return RequestAttributes
+     */
+    protected function getAttributes()
+    {
+        if (!$this->attributes) {
```

Comment:
The problem here is that the `PreviewMessageHandler` uses this method to change the state of the application with different requests. This is very important in the websocket-mode.